pkg/app/piped/planpreview/builder_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package planpreview
+
+import (
+	"testing"
+
+	"github.com/pipe-cd/pipecd/pkg/model"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
to fix a lint error:

```suggestion
	"github.com/stretchr/testify/require"

	"github.com/pipe-cd/pipecd/pkg/model"
)
```

Reply:
Thank you for your review! I appreciate it. 🙏

I've fixed it here: https://github.com/pipe-cd/pipecd/pull/5540/commits/cd13d3bdca88c17ce13bcc8cbf4844318555acc2